backend/src/main/kotlin/blue/mild/covid/vaxx/dao/Question.kt

```diff
@@ -1,9 +1,15 @@
 package blue.mild.covid.vaxx.dao
 
+import org.jetbrains.exposed.sql.Column
 import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.`java-time`.timestamp
+import java.time.Instant
 
 object Question : Table("questions") {
     val id = varchar("id", 36)
+    val created: Column<Instant> = timestamp("created")
```

Comment:
tady to neni treba vubec, protoze tohle bude staticke a nepujde to menit jinak nez pres primy pristup do db